### PR TITLE
Add signature_latte.txt to signature_recipes branch

### PR DESCRIPTION
**Pull request by devvrat2**
Raise a pull request from the branch “signature_recipes” to “master”. Include one of your friends as a Code reviewer adding their GH handle (s) as reviewers.

### CODE REVIEW

File: recipes/signature_latte.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Signature laat khaane ki vidhi
```
Comment (TejaswaVerma) [inline recipes/signature_latte.txt:1]: Dhondu just chill.

**Review by TejaswaVerma — COMMENTED** [review]
Review complete.

File: recipes/signature_latte.txt (hunk 1 above)
Comment (devvrat2) [reply recipes/signature_latte.txt:1]: pann paan

**Comment by TejaswaVerma** [comment]
Part 2

**Pull request merged**